CentOS 7.6.1810 make run ERROR unsatisfiable constraints libffi-3.2.1-r6

Hi All,

make run error message as below, need help

Step 6/15 : RUN apk add --update gcc musl-dev libffi-dev make openssl-dev
 ---> Running in 91a8e6215c0a
fetch http://dl-cdn.alpinelinux.org/alpine/edge/main/x86_64/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/edge/community/x86_64/APKINDEX.tar.gz
ERROR: unsatisfiable constraints:
  libffi-3.2.1-r6:
    breaks: libffi-dev-3.3-r2[libffi=3.3-r2]
    satisfies: .python-rundeps-20200424.014659[so:libffi.so.6]
The command '/bin/sh -c apk add --update gcc musl-dev libffi-dev make openssl-dev' returned a non-zero code: 4
make: *** [build] Error 4


===Environmental information is as follows====
[root@localhost ~]# cat /etc/redhat-release
CentOS Linux release 7.6.1810 (Core)
[root@localhost ~]#
[root@localhost ~]# docker --version
Docker version 19.03.8, build afacb8b
[root@localhost ~]#
[root@localhost ~]# python --version
Python 2.7.5
[root@localhost ~]#

--- a/build/alpine/repositories
+++ b/build/alpine/repositories
@@ -1,2 +1,2 @@
-http://dl-cdn.alpinelinux.org/alpine/edge/main
-http://dl-cdn.alpinelinux.org/alpine/edge/community
+http://dl-cdn.alpinelinux.org/alpine/v3.10/main
+http://dl-cdn.alpinelinux.org/alpine/v3.10/community

--- a/build/alpine/repositories.internal
+++ b/build/alpine/repositories.internal
@@ -1,2 +1,2 @@
-http://repomirror-ict.eng.netapp.com/alpine-linux/latest-stable/main
-http://repomirror-ict.eng.netapp.com/alpine-linux/latest-stable/community
+http://repomirror-ict.eng.netapp.com/alpine-linux/v3.10/main
+http://repomirror-ict.eng.netapp.com/alpine-linux/v3.10/community

